src/CMakeLists.txt

```diff
@@ -90,8 +91,8 @@ endif ()
 if (BUILD_SHARED_LIBS)
   file (MAKE_DIRECTORY "${HDF5_VOL_REST_BINARY_DIR}/shared")
   add_library (${HDF5_VOL_REST_LIBSH_TARGET} SHARED ${HDF5_VOL_REST_SRCS} ${HDF5_VOL_REST_HDRS})
-  TARGET_C_PROPERTIES (${HDF5_VOL_REST_LIBSH_TARGET} SHARED " " " ")
-  target_link_libraries (${HDF5_VOL_REST_LIBSH_TARGET} PUBLIC ${HDF5_LIBRARIES_TO_EXPORT} ${LINK_SHARED_LIBS})
+  RV_TARGET_C_PROPERTIES (${HDF5_VOL_REST_LIBSH_TARGET} SHARED " " " ")
+  target_link_libraries (${HDF5_VOL_REST_LIBSH_TARGET} PUBLIC ${HDF5_LIBRARIES_TO_EXPORT} ${LINK_SHARED_LIBS} ${HDF5_LIBRARIES})
```

Comment:
We should also probably replace all usages of `HDF5_LIBRARIES_TO_EXPORT` with `HDF5_VOL_REST_LIBRARIES_TO_EXPORT` in this project.